docs/docs/user-guide/using-cli.mdx

```diff
@@ -237,3 +239,47 @@ To help you use its commands, CLI for Microsoft 365 offers you the ability to au
 ## Disable automatic checking for updates
 
 Each time you run CLI for Microsoft 365, it will automatically check if there is a new version available and prompt you with update instructions if that's the case. If you use CLI for Microsoft 365 in CI/CD or in scripts and want to make it run faster, you can disable the check by setting the `CLIMICROSOFT365_NOUPDATE` environment variable to `1`.
+
+## Using CLI for Microsoft 365 in PowerShell
+
+CLI for Microsoft 365 by default outputs errors to stderr. Unfrortunately, PowerShell doesn't handle stderr as well as other shells. Also in case of command errors PowerShell does not break execution and may still continue even after setting `$ErrorActionPreference = "Stop"`. In order to overcome those problems you may use the following settings and helper method.
+
+```PowerShell
+  m365 cli config set --key output --value json # setting output to json
+  m365 cli config set --key errorOutput --value stdout # changing error output to stdout
+  m365 cli config set --key showHelpOnFailure --value false # don't show help on command failure
```

Comment:
Is this necessary for use in PowerShell or a personal preference?

Reply:
it is necessary:
- we need the error message to be part of stdout not stderr
- we need the output to be json format always since we `| ConvertFrom-Json` the output every time
- the aim of this guide is to provide a helper method to use CLI in PS script. In case of script execution showing help information on failure does not make sense and if the script saves logs it just brings more confusion in the log file which is not needed.